bin/dynamic_challenge_score.py

```diff
@@ -272,7 +277,23 @@ def reconstruction(truth: np.ndarray, prediction: np.ndarray) -> float:
     return e1
 
 
-# TODO: Not final, update once organizers confirm all inputs and metrics
+def HouseZeroScore(truth: np.ndarray, prediction: np.ndarray) -> Tuple[float]:
+    '''Produce errors for the HouseZero model.'''
+    # error for room temperature
```

Comment:
nit: It might be a bit easier to name the variable this instead of adding an extra comment. Like:
`error_room_temperature = 100 * ...`